XenAdmin/Wizards/NewVMWizard/NewVMWizard.cs

```diff
@@ -173,6 +173,7 @@ protected override void FinishWizard()
                                         (long)page_5_CpuMem.SelectedMemoryDynamicMin,
                                         (long)page_5_CpuMem.SelectedMemoryDynamicMax,
                                         (long)page_5_CpuMem.SelectedMemoryStaticMax,
+                                        page_5_CpuMem.SelectedBootMode,

```

Comment:
This new functionality has been added to the Cpu and Memory page according to the design, but I have two issues with it:
1) We seem to have space issues on this page. I suppose we could reduce a bit the margins of the existing controls, but if the bottom warning appears, we'll have truncation issues.
2) I don't think the boot mode matches the rest of the content of this page; the Installation media is more relevant.

I'm going to place a request to reconsider this design. Could you please hold on a bit on this comment until I confirm this with all involved stakeholders.

Reply:
Tina, thanks very much for the comments.

I agree that boot mode should not be with CPU & Memory.
And the similar design also happens in CP-28679, so could you please also involve it in your design reconsideration request?

Reply:
Hmmm, the ImportWizard is slightly different in that it is more compact than the NewVM wizard and already the VM definition page contains controls from across 3 different pages of the NewVM wizard (Name, CPU-Memory, and Disk). For the time being, and before considering an extra page, let's see first if we can fit the boot options above the .wim option (we may need to remove some of the empty gridlines or other empty space this page has).
Also, one thing you may want to consider is moving the boot options on a custom control which could be used as is on both wizards.

Reply:
It seems that there is not enough space above the .wim option, but let me have a try.
I agree to extract the boot options as a UserControl, and I will do that, thank you.